src/main/java/net/magik6k/mpt/db/RepoBase.java

```diff
@@ -110,4 +110,17 @@ public int getUserRepoCount(String user){
 		}
 		return res;
 	}
+
+	public List<String> getAllRepos(){
+		List<String> res = new LinkedList<String>();
+		for(DBObject repo: repos.find()){
+			if(repo.containsField("users")){
+				Object userList = repo.get("users");
+				for(Object name : (BasicDBList) userList){
```

Comment:
A repo can have multiple users, so it may be added to the list multiple times, is that expected?